js/hathi-trust-availability.module.js

```diff
@@ -85,15 +85,26 @@ angular
 
       self.$onInit = function() {
         setDefaults();
-        if (!(isOnline() && self.hideOnline)) {
-          updateHathiTrustAvailability();
-        }
+
+        // prevent appearance/request iff 'hide-online' 
+        if (isOnline() && self.hideOnline) { return; }
+
+        // prevent appearance/request iff 'hide-if-journal'
+        if(isJournal() && self.hideIfJournal){ return; }
```

Comment:
instead of stacking the conditionals before the call, i've separated them and added an empty `return`.